### PR TITLE
Update gradio_helper.py to fix processor input issues for mllama-3.2

### PR DESCRIPTION
**Pull request by zhuo-yoyowz**
Run notebook and met the same issue as described in https://github.com/meta-llama/llama-recipes/issues/695.
Fixed the issue using fix here https://github.com/meta-llama/llama-recipes/pull/696/files.

### CODE REVIEW

**Comment by eaidova** [comment]
Thanks, @zhuo-yoyowz could you please provide info about transformers version which you use? I believe these changes may be unsafe and nonbackward compatible as on my end gradio worked well on transformers 4.45.0

File: notebooks/mllama-3.2/gradio_helper.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def bot_streaming(message, history):
         prompt = processor.tokenizer.apply_chat_template(conversation, tokenize=False, add_generation_prompt=True)
         print(f"prompt is -\n{prompt}")
         image = Image.open(image)
-        inputs = processor(prompt, image, return_tensors="pt")
+        inputs = processor(image, prompt, return_tensors="pt")
```
Comment (eaidova) [inline notebooks/mllama-3.2/gradio_helper.py]: I suppose this fix will be more reliable:
```suggestion
        inputs = processor(images=image, text=prompt, return_tensors="pt")
```

**Comment by zhuo-yoyowz** [comment]
> Thanks, @zhuo-yoyowz could you please provide info about transformers version which you use? I believe these changes may be unsafe and nonbackward compatible as on my end gradio worked well on transformers 4.45.0

I used transformers 4.45.1, but I also tried 4.45.0 but failed. Not sure where the issue comes from.

File: notebooks/mllama-3.2/gradio_helper.py (hunk 1 above)
Comment (zhuo-yoyowz) [reply notebooks/mllama-3.2/gradio_helper.py]: > I suppose this fix will be more reliable:

Agree. This one looks good. Thank you!

**Review by eaidova — APPROVED** [review]

**Pull request merged**